.github/workflows/tests-nightly.yml

```diff
@@ -313,6 +315,8 @@ jobs:
           API_URI: ${{ needs.get-environment-variables.outputs.url }}graphql/
           APP_MOUNT_URI: ${{ secrets.APP_MOUNT_URI }}
           BASE_URL: ${{ needs.get-environment-variables.outputs.url }}dashboard/
+          ## backward compatibility requires CYPRESS_baseUrl as env var name - can be removed when version lower than 3.17 are no longer supported
+          CYPRESS_baseUrl: ${{ needs.get-environment-variables.outputs.url }}dashboard/
```

Comment:
it's only mandatory here, everywhere else you can remove it. Workflow needs to have BC only for repository_dispatch